src/pages/Explore/redirects.tsx

```diff
@@ -1,12 +1,29 @@
 import { Navigate, useParams } from 'react-router-dom'
 
-export default function RedirectExploreTokens() {
-  const { chainName, tokenAddress } = useParams<{ chainName: string; tokenAddress: string }>()
-  if (chainName && tokenAddress) {
+import Explore, { ExploreTab } from '.'
+
+// useParams struggles to distinguish between /explore/:chainId and /explore/:tab
+export function useExploreParams() {
+  const { tab, chainName, tokenAddress } = useParams<{ tab: string; chainName: string; tokenAddress: string }>()
+  const exploreTabs = Object.values(ExploreTab)
+  if (!tab && chainName && exploreTabs.includes(chainName as ExploreTab)) {
+    return { tab: chainName, undefined, tokenAddress }
```

Comment:
shouldn't the tab be `tokens` and still return the `chainName` in this case?

Reply:
I think this might be causing an error I'm seeing, routing to `https://interface-git-info-explore-page-uniswap.vercel.app/explore/arbitrum` is giving me the explore token page for Ethereum

Reply:
correct link should be `https://interface-git-info-explore-page-uniswap.vercel.app/explore/tokens/arbitrum`! I don't think a user would ever get linked to /explore/arbitrum unless they manually type it in

Reply:
https://github.com/Uniswap/interface/blob/info/explore-page/src/pages/RouteDefinitions.tsx#L107-L108